package-lock.json

```diff
@@ -3189,6 +3189,7 @@
           "dev": true,
           "optional": true,
           "requires": {
+            "bindings": "^1.5.0",
```

Comment:
rollback this file, if you do not change package.json

Reply:
done